snuba/web/split.py

```diff
@@ -128,9 +173,51 @@ def time_split(dataset, request: Request, *args, **kwargs):
 
         return overall_result
 
-    def col_split(
-        dataset, request: Request, column_split_spec: ColumnSplitSpec, *args, **kwargs
-    ):
+
+class ColumnSplitSpec(NamedTuple):
+    """
+    Provides the column names needed to perform column splitting.
+    id_column represent the identity of the row.
+
+    """
+
+    id_column: str
+    project_column: str
+    timestamp_column: str
+
+    def get_min_columns(self) -> Sequence[str]:
+        return [self.id_column, self.project_column, self.timestamp_column]
```

Comment:
It seems like this data structure is just used as a constructor parameter to `ColumnSplitQueryStrategy` at this point. Should these just be instance attributes on that class now? Or is there other benefit to keeping this around?

Reply:
There is no benefit in keeping it around. In this case as well I kept it only not to touch how the splitter worked internally and focus on the change to the code structure to make the change in the PR more evident. 
Though I am ok removing it right away.